docs/photoniq/vwrs/02-get-started-vwrs/get-started-vwrs-web-and-mobile.md

```diff
@@ -0,0 +1,85 @@
+
+3. [Create and configure your waiting room](index.md): This step allows you to fine-tune how the waiting room operates and integrates with your site.
+
+## Integrating on Mobile
+
+Mobile apps, also called native apps or system-specific apps, are developed for specific platforms using platform-specific programming languages. Unlike web-based apps, mobile apps are developed for specific platforms using platform-specific UI frameworks and the application takes on a greater role in managing the user experience during high-traffic periods, including handling the display of waiting times and managing user expectations. Mobile apps require direct API interactions to manage user experiences effectively during high traffic periods.
+
+### Steps
+
+1. [Create your API Key](https://www.macrometa.com/docs/apiVwrs#/operations/createAPIKey): Start by creating an API key to access the VWRs system, as this key is essential for all API interactions.
+
+2. Create and Configure your waiting room
+
+3. Use the [VWRs API](https://www.macrometa.com/docs/apiVwrs#/) to observe and manage the user's position in the waiting room. This involves making periodic API calls to check the queue status and manage the user experience accordingly.
+
+  - [Get initial position in waiting room](https://www.macrometa.com/docs/apiVwrs#/paths/api-vwr-v1-position-domain_key/get)
+  - [Get current position in waiting room](https://www.macrometa.com/docs/apiVwrs#/paths/api-vwr-v1-position-domain_key---request_id/get)
+
+### Pseudo-Code Example
+
+Here is a pseudo-code example illustrating how a mobile app interacts with the VWR API to manage a user's waiting experience:
+
+```javascript
+// Initial request specifying the waiting room.
+let waitingRoom = "shopping";
+let result = GET("/api/vwr/v1/position/" + waitingRoom);
+
+// Loop while the user is still in the waiting room.
+while (result.position > 0) {
+    // Notify the user of their current position and expected wait time.
+    notifyUser(result.position, result.waiting_time);
+
+    // Wait for the specified refresh interval before checking the position again.
+    sleep(result.refresh_interval);
+
+    // Re-check the user's position using the visitor_id received from the initial call.
+    result = GET("/api/vwr/v1/position/" + waiting_room + "/" + result.visitor_id);
+}
+
+// Proceed with accessing the protected resource once the user is out of the waiting room.
+accessResource();
+```
+
+## Next Steps
+
+Once your virtual waiting room is operational, consider monitoring and managing its performance to ensure optimal functionality:
+
+- **Metrics and Analytics:** Regularly check the [VWRs metrics](../05-monitoring-vwrs/index.md) to assess the performance and effectiveness of your waiting room.
+- **Review Usage Patterns:** Analyze usage data to optimize settings and improve user experiences. You can customize your settings to analyze usage statistics at different time intervals:
```

Comment:
is that "... to analyze usage statistics" or "... by analyzing usage statistics"